draft-kuehlewind-iab-rfc4053bis.md

```diff
@@ -227,46 +227,36 @@ from other organisations.
    another body; if it fails to react in a timely fashion, it may miss
    the opportunity.
 
-#  Lifetime of a Liaison Statement
+#  Recording Liaison Statements
 
-   A liaison statement is a temporary document, much like an internet
-   draft.  If it affects IETF output, the normal expectation is that the
-   resulting RFC will contain relevant information that remains
-   pertinent.  Retaining liaison statements that have been completely
-   dealt with mostly serves to hide new ones and create the appearance
-   of not dealing with them.
+For the IETF, a liaison statement is a message that was sent or received (usually an email or some kind of formal letter)
+that is recorded in our liaison management tool. i.e. the value of sending a liaison statement for an organization compared to a mail,
+is that it will officially be recorded and the public record will attest that certain information has been communicated between the organizations.
```

Comment:
```suggestion
For the IETF, a liaison statement is a message that was sent or received
(usually an email or some kind of formal letter)
that is recorded in our liaison management tool,
i.e. the value of sending a liaison statement for an organization compared to a mail,
is that it will officially be recorded and the public record will attest
that certain information has been communicated between the organizations.
```